e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
```

Comment:
I would remove this too

Reply:
Cypress test runner does not work without the support file, we get the following error when removing it:

![Screen Shot 2022-11-27 at 2 06 21 PM](https://user-images.githubusercontent.com/50315144/204162347-dc9d9301-e212-4f6c-a13f-bac718d2b446.png)

Reply:
We could set it to `false`, but it feels nice to have for the future since it allows you to create custom commands. Similar to how we created login logout commands in the loveloom application to prevent having to move through the different routes and user interactions to get to the apps home page in each test.

Reply:
Ok that's fine, I would personally choose to remove, but don't mind it, so all good